examples/starrocks/deploy_a_starrocks_cluster_with_cn.yaml

```diff
@@ -0,0 +1,64 @@
+# This manifest deploys a StarRocks cluster with FE, BE, CN.
```

Comment:
can describe the cluster structure, e.g. with 3 FEs, 3 BEs, and CN with auto scaling enabled.